.github/workflows/main.yml

```diff
@@ -84,9 +84,8 @@ jobs:
       - uses: actions/checkout@v4
       - name: Install packages
         run: |
-          sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
           sudo apt-get update
-          sudo apt-get install -y ninja-build gcc-13 g++-13 clang-15
+          sudo apt-get install -y ninja-build gcc-12 g++-12 clang-15
```

Comment:
isn't it installed by default?
```suggestion
          sudo apt-get install -y ninja-build clang-15
```

Reply:
No the default is `gcc-11`.

Reply:
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md

Reply:
interesting, they must've changed that

Reply:
no actually, because we set `CC=gcc-12 CXX=g++-12`, it should work just fine. They are installed by default, just not added as the default `CC` and `CXX` compilers

Reply:
fine with keeping it as is though. makes stuff explicit

Reply:
In the readme of images `gcc-12` is not listed as installed.

Reply:
![image](https://github.com/PowerGridModel/power-grid-model/assets/15234327/3d8db063-980b-4c4f-b175-5c16d8b42727)